.github/workflows/builder-release.yaml

```diff
@@ -0,0 +1,43 @@
+name: Builder - Release
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Releases Repo
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          fetch-depth: 0
+      - run: ls -la
+      - name: Push cmd/builder Tag
```

Comment:
Will this have any effect on the release as-is? This workflow runs on the normal `vX.Y.Z` tag and also does the release. My first guess is that GoReleaser will use the tag for the current branch as the release tag.

Reply:
[goreleaser lets you specify which tags to use on a build/release job using environment variables](https://goreleaser.com/cookbooks/set-a-custom-git-tag/), specifically referencing the situation in which one commit has multiple tags; this might be something worthwhile to do in order to be safe. I can update the release-contrib, release-core, and release-k8s workflow as well as the builder-release workflow to keep the tags separated via setting the environment variable

Reply:
of course, if I were to refactor the builder release to join the "partial" build chain that core, contrib, and k8s form under the vX.YYY.Z tag instead of cmd/builder/vX.YYY.Z, this wouldn't be necessary

Reply:
I've confirmed in my own repo that they do in fact build and release separately, successfully. See here: https://github.com/jackgopack4/opentelemetry-collector-releases/releases